### PR TITLE
V2 Feature - TRSV2-206 - V2 registration journey

### PR DESCRIPTION
**Pull request by chris-pettinga**

### CODE REVIEW

File: trade_remedies_public/templates/v2/component_macros/checkbox.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+                <h1 class="govuk-fieldset__heading">
+                    Terms of Use and Privacy Notice
+                </h1>
+            </legend>
+            {% if error %}
+                <p id="{{ name }}-error" class="govuk-error-message">
+                    {% for error in error %}
+                        <span class="govuk-visually-hidden">Error:</span> {{ error }}
```
Comment (KnowYourLines) [inline trade_remedies_public/templates/v2/component_macros/checkbox.html:14]: Do these need to be updated for multiple errors being returned from the API?

**Review by KnowYourLines — CHANGES_REQUESTED** [review]

**Review by holdenweb — APPROVED** [review]
Approved after review module a few small changes.

File: trade_remedies_public/templates/v2/registration/registration_your_uk_employer.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            source: function (query, populateResults) {
+                $.ajax
+                ({
+                    type: "GET",
+                    url: `{% url 'companieshouse' %}?term=${query}`,
+                    success: function (data) {
+                        if (data) {
+                            let names = data.map(result => `${result.title} (${result.company_number})`)
+                            proposed_names = Object.fromEntries(data.map(result => [`${result.title} (${result.company_number})`, result]));
+                            populateResults(names)
```
[the remark this answers is not in the document]
Comment (chris-pettinga) [reply trade_remedies_public/templates/v2/registration/registration_your_uk_employer.html:77]: populateResults is a method exposed by accessibleAutocomplete library imported on line 59

[the remark this answers is not in the document]
Comment (KnowYourLines) [reply trade_remedies_public/templates/v2/registration/registration_your_uk_employer.html:77]: Yes thought so, but I can't find any mention of it when doing CTRL+F in `trade_remedies_public/templates/static/v2/js/accessible-autocomplete.min.js`

File: trade_remedies_public/templates/v2/registration/registration_set_password.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    </ul>
+    {% include "v2/component_macros/password_field_with_button.html" with autocomplete="new_password" %}
+    {% include "v2/component_macros/form_button_group.html" with forward_button_text="Continue" %}
+{% endblock form_content %}
+
+{% block scripts %}
+    <script src="{% static 'v2/js/password_input.js' %}"></script>
+    <script type="text/javascript">
+
+        $("#password").keyup(function () {
```
[the remark this answers is not in the document]
Comment (chris-pettinga) [reply trade_remedies_public/templates/v2/registration/registration_set_password.html:33]: You're right thanks, fixed

File: trade_remedies_public/templates/v2/component_macros/checkbox.html (hunk 1 above)
Comment (chris-pettinga) [reply trade_remedies_public/templates/v2/component_macros/checkbox.html:14]: Fields with multiple errors in a list are only returned as part of the error summary box at the top of the page, not for individual errors

**Pull request merged**